### PR TITLE
Fix deprecated polyfill issue

### PR DESCRIPTION
**Pull request by parvez**
Closes https://github.com/danger/danger-js/issues/857

### CODE REVIEW

**Comment by parvez** [comment]
Following through the closed PR https://github.com/danger/danger-js/pull/928, that required some additional fixes. This PR should help remove the deprecation notice to users of this node module! 🎉

File: package.json [hunk 1]
```diff
@@ -133,6 +134,7 @@
     "nock": "^10.0.6",
     "pkg": "^4.4.2",
     "prettier": "^2.5.1",
+    "regenerator-runtime": "^0.13.9",
```
Comment (orta) [inline package.json:137]: I think this probably needs to be in the dependencies section

Comment (parvez) [reply package.json]: yes. moved them to dependencies!

**Comment by orta** [comment]
Thanks!

**Comment by orta** [comment]
Shipped in https://github.com/danger/danger-js/releases/tag/11.0.4

**Pull request merged**